### ISSUE TITLE
Feature Request - Multiple .CSV per chart/report

### ISSUE THREAD

**Issue by StrayaJake** [yet-another-tool/obsidian-findoc#8]
Would be great if we could have multiple .CSV per chart

IE, Income and expenses, could be separate .CSV

```findoc
filename: ./Items/Income.csv, ./Items/Expenses.csv
model: all
type: chart
```

**Comment by studiowebux** [comment]
Thanks ! 

Yes ! It will help a lot when having a lot of data to ingest and manage, 

I will regroup a couple of Feature requests and try to release soon

**Comment by studiowebux** [comment]
Hello
I released v0.5.3 and 0.5.4 that includes this feature, and started to work on v0.6.0 and tested the feature successfully.

Note, Each file must have the header (I skip the first line to join all files together)

You can update the plugin in Obsidian, 
I will close this ticket, do not hesitate to reopen if you have comments
